pkg/policy/l4.go

```diff
@@ -220,30 +220,27 @@ func (a *PerSelectorPolicy) Equal(b *PerSelectorPolicy) bool {
 type AuthType uint8
 
 const (
-	// AuthTypeNone means no authentication required
-	AuthTypeNone AuthType = iota
-	// AuthTypeNull is a simple auth type that always succeeds
-	AuthTypeNull
-	// AuthTypeMTLSSpiffe is a mTLS auth type that uses SPIFFE identities with a SPIRE server
-	AuthTypeMTLSSpiffe
+	// AuthTypeDisabled means no authentication required
+	AuthTypeDisabled AuthType = iota
+	// AuthTypeAuthSpire is a mutual auth type that uses SPIFFE identities with a SPIRE server
+	AuthTypeAuthSpire
```

Comment:
why having `Auth` twice in the const name? 

-> `AuthTypeSpire`?

Reply:
good catch :100: